{"variants":[{"paths":["\/documentation\/scanbotsdk\/sbsdkimageprocessorerror\/writeerror"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError\/writeError"},"metadata":{"externalID":"s:10ScanbotSDK24SBSDKImageProcessorErrorO05writeE0yA2CmF","roleHeading":"Case","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"writeError","kind":"identifier"}],"symbolKind":"case","role":"symbol","title":"SBSDKImageProcessorError.writeError","modules":[{"name":"ScanbotSDK"}]},"hierarchy":{"paths":[["doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK","doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError"]]},"schemaVersion":{"patch":0,"major":0,"minor":3},"kind":"symbol","abstract":[{"text":"An unknown error occured while writing an image to a file.","type":"text"}],"sections":[],"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"writeError"}]}],"kind":"declarations"}],"references":{"doc://net.doo.ScanbotSDK/documentation/ScanbotSDK":{"role":"collection","title":"ScanbotSDK","url":"\/documentation\/scanbotsdk","identifier":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK","type":"topic","kind":"symbol","abstract":[{"text":"Version: 6.0.0","type":"text"}]},"doc://net.doo.ScanbotSDK/documentation/ScanbotSDK/SBSDKImageProcessorError":{"title":"SBSDKImageProcessorError","abstract":[{"type":"text","text":"An enum of possible errors when processing images."}],"type":"topic","kind":"symbol","role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"SBSDKImageProcessorError"}],"navigatorTitle":[{"kind":"identifier","text":"SBSDKImageProcessorError"}],"url":"\/documentation\/scanbotsdk\/sbsdkimageprocessorerror","identifier":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError"},"doc://net.doo.ScanbotSDK/documentation/ScanbotSDK/SBSDKImageProcessorError/writeError":{"title":"SBSDKImageProcessorError.writeError","abstract":[{"type":"text","text":"An unknown error occured while writing an image to a file."}],"type":"topic","kind":"symbol","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"writeError"}],"url":"\/documentation\/scanbotsdk\/sbsdkimageprocessorerror\/writeerror","identifier":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError\/writeError"}}}